orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -682,6 +693,45 @@ at::Tensor& out) {
   return out;
 }
 
+// aten::abs.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+at::Tensor& abs_out(
+  const at::Tensor& self,
+  // *,
+  at::Tensor& out) {
```

Comment:
when aten invoke "*.out" ops, is the output tensor already allocated before the invoke? if that's the case, we shouldn't internally allocate a temporary tensor and copied in.

Reply:
If I understand correctly - you are suggesting that instead of this:

```
auto tmp = aten_tensor_from_ort(
    std::move(ort_outputs_0_Abs[0]),
    tensor_options);
```

we could just copy the ort output directly to the output tensor? I think this seems reasonable, I will look into it.